### PR TITLE
core-node-docs: upd-in

### PR DESCRIPTION
**Pull request by sumitshinde-84**
## Checklist

 - [ ] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

File: src/_includes/core-nodes/udp-in-use-case.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+## What is udp-in node in Node-RED?
+
+The UDP-In node in Node-RED enables the reception of UDP messages from remote devices or services. It acts as a listener, waiting for incoming UDP packets on a specified port. This functionality is crucial for real-time applications, such as IoT data ingestion and network communication. Whether you're receiving sensor data from IoT devices or communicating with other networked systems, the UDP-In node seamlessly integrates UDP communication into your Node-RED workflows, providing a lightweight and efficient solution for data reception.
+
+### What is Udp?
+
+UDP (User Datagram Protocol) is a connectionless protocol in the Internet Protocol suite. It transmits data packets, or datagrams, without establishing a connection, prioritizing speed over reliability. Commonly used for real-time applications like video streaming, online gaming, and VoIP. UDP's simplicity reduces latency but doesn't guarantee delivery of data packets. For more information refer to [UDP MDN Docs](https://developer.mozilla.org/en-US/docs/Glossary/UDP)
+
+## Configuring UDP-In Node
```
Comment (ZJvandeWeg) [inline src/_includes/core-nodes/udp-in-use-case.md:9]: Have you tried this on FlowFuse?

Comment (sumitshinde-84) [reply src/_includes/core-nodes/udp-in-use-case.md:9]: No, I haven't tried it on the FlowFuse cloud; I've tried it on the locally running Node-RED instance.

Comment (ZJvandeWeg) [reply src/_includes/core-nodes/udp-in-use-case.md:9]: Please update the documentation with a line that explains why these aren't available on FFC

File: src/_includes/core-nodes/udp-in-use-case.md [hunk 2]
```diff
@@ -29,6 +29,8 @@ The UDP-In node in Node-RED provides versatile configuration options to tailor U
 
 *Note: On some systems, you may need root or administrator access to use ports below 1024 and/or broadcast, and have to ensure your firewall allows the data in.*
 
+**Note: The default TCP and UDP nodes have been removed from the Node-RED palette in the FlowFuse Cloud due to limitations in routing connections to the container running Node-RED inside the FlowFuse platform**
```
Comment (ZJvandeWeg) [inline src/_includes/core-nodes/udp-in-use-case.md]:
```suggestion
**Note: The default TCP and UDP nodes have been removed from the Node-RED palette in FlowFuse Cloud due to limitations in routing connections to the container running Node-RED inside the FlowFuse platform**
```

File: src/_includes/core-nodes/udp-in-use-case.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
+## What is udp-in node in Node-RED?
+
+The UDP-In node in Node-RED enables the reception of UDP messages from remote devices or services. It acts as a listener, waiting for incoming UDP packets on a specified port. This functionality is crucial for real-time applications, such as IoT data ingestion and network communication. Whether you're receiving sensor data from IoT devices or communicating with other networked systems, the UDP-In node seamlessly integrates UDP communication into your Node-RED workflows, providing a lightweight and efficient solution for data reception.
+
+### What is Udp?
```
Comment (ZJvandeWeg) [inline src/_includes/core-nodes/udp-in-use-case.md:5]:
```suggestion
### What is UDP?
```

File: src/_includes/core-nodes/udp-in-use-case.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+UDP (User Datagram Protocol) is a connectionless protocol in the Internet Protocol suite. It transmits data packets, or datagrams, without establishing a connection, prioritizing speed over reliability. Commonly used for real-time applications like video streaming, online gaming, and VoIP. UDP's simplicity reduces latency but doesn't guarantee delivery of data packets. For more information refer to [UDP MDN Docs](https://developer.mozilla.org/en-US/docs/Glossary/UDP)
```
Comment (ZJvandeWeg) [inline src/_includes/core-nodes/udp-in-use-case.md:7]:
```suggestion
UDP (User Datagram Protocol) is a connectionless protocol in the Internet Protocol suite. It transmits data packets, or datagrams, without establishing a connection, prioritizing speed over reliability. Commonly used for real-time applications like video streaming, online gaming, and VoIP. UDP's simplicity reduces latency but doesn't guarantee delivery of data packets. For more information refer to [UDP MDN Docs](https://developer.mozilla.org/en-US/docs/Glossary/UDP).
```

**Review by ZJvandeWeg — APPROVED** [review]

**Pull request merged**